cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/readJsonFile/SlaReader.java

```diff
@@ -24,14 +24,11 @@
 import com.google.gson.Gson;
 import java.io.FileNotFoundException;
 import java.io.FileReader;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
 
 /**
  * This class read the sla agreements in json format.
  * 
- * The sla agreements is in the {@link SlaContractMetrics}. This class 
+ * The sla agreements is in the {@link DimensionMetricsSlaContract}. This class 
```

Comment:
The class DimensionMetricsSlaContract was changed but reference in the link tag was not changed. Update your NetBeans to the latest version that probably will reduce such problems.